cluster-operator/src/main/resources/entityTopicOperatorDefaultLoggingProperties

```diff
@@ -5,9 +5,7 @@ appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = [%d] %-5p <%-12.12c{1}:%L> [%-12.12t] %m%n
 
-property.entity-topic-operator.root.logger=${env:STRIMZI_LOG_LEVEL:-INFO}
-
-rootLogger.level = ${entity-topic-operator.root.logger}
+rootLogger.level = INFO
```

Comment:
Can you explain how this file gets used with the one from the CM, because it's not clear to me. If might be worth adding a comment to the file to explain it too.

Reply:
This default file is used to load default settings. Additionally, new setting is parsed and the CM is updated. From this updated CM `log4j2.properties` file is created (method `getVolumes()`). When the TO or UO is deployed, it has an option `-Dlog4j2.configurationFile` in the `run.sh` script, which is set path to the created file with the updated settings.